eng/helix/content/.runsettings

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RunSettings>
+  <RunConfiguration>
+  </RunConfiguration>
+  <LoggerRunSettings>
+    <Loggers>
+      <Logger friendlyName="xunit" />
+      <Logger friendlyName="console">
+        <Configuration>
+          <Verbosity>Normal</Verbosity>
+        </Configuration>
+      </Logger>
+    </Loggers>
+  </LoggerRunSettings>
+  <DataCollectionRunSettings>
+    <DataCollectors>
+      <DataCollector friendlyName="blame" enabled="True">
+        <Configuration>
+          <CollectDump DumpType="Full" />
+          <CollectDumpOnTestSessionHang TestTimeout="10min" HangDumpType="Full" />
```

Comment:
Why double the timeout❔

Reply:
The 5 minutes i picked was arbitrary before, seems better to just use what runtime is using in their settings as a default.

Reply:
Oh, nevermind. That's the overall job timeout less 5 minutes.